### PR TITLE
Rails 4 upgrade

### PR DESCRIPTION
**Pull request by tboerger**
Just some fixes that are included for the rails 4 update at https://github.com/crowbar/barclamp-crowbar/pull/1157 for getting feedback.

### CODE REVIEW

**Comment by tboerger** [comment]
This merge is missing :)

**Comment by vuntz** [comment]
> This merge is missing :)

Because of the FIXME. We'll use this as a patch in the packages, but this won't get merged with the FIXME.

File: chef/cookbooks/ohai/files/default/plugins/crowbar.rb [hunk 1]
```diff
@@ -80,6 +80,9 @@ class EthtoolValue < CStruct
 
 def get_supported_speeds(interface)
   begin
+    # This raise hides segfault on ifreq.unpack. The rescue will ensure we
+    # return the defaults.
+    raise "FIXME"
```
Comment (dirkmueller) [inline chef/cookbooks/ohai/files/default/plugins/crowbar.rb]: this has been fixed in our ruby package. please revert the patch.

Comment (tboerger) [inline chef/cookbooks/ohai/files/default/plugins/crowbar.rb]: @bkutil Can you please remove that temporary "fix"?

**Comment by bkutil** [comment]
@dirkmueller I took your word for it and removed the fixme's.

**Comment by tboerger** [comment]
:+1:

**Pull request merged**